Sources/Yams/Representer.swift

```diff
@@ -18,10 +18,14 @@ private let cpow: (_: Double, _: Double) -> Double = ucrt.pow
 import CoreFoundation
 import Bionic
 private let cpow: (_: Double, _: Double) -> Double = Bionic.pow
-#else
+#elseif canImport(Glibc)
 import CoreFoundation
 import Glibc
 private let cpow: (_: Double, _: Double) -> Double = Glibc.pow
+#elseif canImport(Musl)
+import CoreFoundation
+import Musl
+private let cpow: (_: Double, _: Double) -> Double = Musl.pow
```

Comment:
Looking at the background to the introduction of this, it now seems like it would be okay to use STL math functions. In that case, we would need to check that there are no problems with all toolchains. If we are only going to support Musl, I think it would have less of an impact if you just explicitly loaded cpow.

introduce commit: https://github.com/jpsim/Yams/commit/682d4985e3c8af8b5b7239fd9436ab9734b4d775

commit message:
```
We must disambiguate `Double.pow(_: Double, _: Double) -> Double` and
`__C.pow(_: Double, _: Double) -> Double` as the Swift for TensorFlow
branch adds the former into the standard library.  The explicit module
dispatch ensures that we make the overload resolution unambiguous
allowing building Yams.
```

Reply:
As far as I can tell, this workaround was only to support some of the low level/standard library changes as part of [Swift for Tensorflow](https://github.com/tensorflow/swift). As that project is now archived (and has been for several years at this point), I think reverting these Tensorflow-specific workarounds is fine and probably desirable so we don't run into these issues with other platforms/libc. Changes in #436.